### DISCUSSION TITLE
TypeError('can only concatenate str (not "int") to str')

### DISCUSSION THREAD

**Discussion by Yukino12333** [Qexo/Qexo#202] [Q&A]
![IMG_20220814_015149.jpg](https://user-images.githubusercontent.com/99794710/184505276-bb8cd7fc-0e30-4e13-a771-b354364918ea.jpg)
咋回事呢

**Comment by am-abudu** [comment]
Bug: 无法正确解析数字作为主题名

将在下一个版本/Dev分支修复
